nrel/routee/powertrain/trainers/ngboost_trainer.py

```diff
@@ -0,0 +1,63 @@
+from enum import Enum
+import pandas as pd
+
+from ngboost import NGBRegressor
+from ngboost.distns import Exponential, Normal
+
+
+from nrel.routee.powertrain.core.model_config import ModelConfig
+from nrel.routee.powertrain.estimators.estimator_interface import Estimator
+from nrel.routee.powertrain.estimators.onnx import ONNX_INPUT_NAME, ONNXEstimator
+from estimators.ngboost_estimator import NGBoostEstimator
+from nrel.routee.powertrain.trainers.trainer import Trainer
+
+
+class NGBoostTrainer(Trainer):
+    def __init__(
+        self,
+        # max_depth: int = 10,
+        # min_samples_split: int = 10,
+        n_estimators: int = 100,
+        Dist=Normal,
```

Comment:
can you adjust this `Dist` parameter to be lower case `dist` to match python naming conventions?